lib/fs/include/lib/fs.h

```diff
@@ -59,12 +70,17 @@ status_t fs_open_dir(const char *path, dirhandle **handle) __NONNULL();
 status_t fs_read_dir(dirhandle *handle, struct dirent *ent) __NONNULL();
 status_t fs_close_dir(dirhandle *handle) __NONNULL();
 
+status_t fs_stat_fs(const char* mountpoint, struct fs_stat* stat);
+
 /* convenience routines */
 ssize_t fs_load_file(const char *path, void *ptr, size_t maxlen) __NONNULL();
 
 /* walk through a path string, removing duplicate path seperators, flattening . and .. references */
 void fs_normalize_path(char *path) __NONNULL();
 
+/* Remove any leading spaces */
```

Comment:
doesn't really match what the function does, which only removes leading slashes.